data/translations/ja-JP.yaml

```diff
@@ -25,10 +25,10 @@ topsectioncus: お問い合わせ
 topsectionwhy: なぜイソリアか？
 topsectionwwd: 主なソリューション
 topsectionwwdtext: イソリアのプロフェショナルスタッフはインフラから、オペレーション、開発、マネージメントとガバナンスまで幅広い範囲にわたるIT活動に対応いたします。貴社の指令、規格と委任に基づいて、貴社が必要とされるニーズに必ずお応えいたします。
-topslogan: イソリアにお任せ下さい
-topsubslogan: 貴社の頼れるIT部となります
-toptestimoninal1: イソリアが我々の複雑で、複数年に及ぶプロジェクトの管理を成功に導いた時、まるで長いトンネルの先に希望の光が差し込んだような状況でした。
-toptestimoninal2: イソリアの技術的な知識は誰にも劣らないうえに、誰でも詳細が理解できるように一般的な用語で複雑な解決方案を説明いただきました。
+topslogan:　イソリアにお任せください！
+topsubslogan: 貴社の頼れるIT部門となります
+toptestimoninal1: イソリアは我々の複雑かつ複数年に及ぶプロジェクトを成功に導いてくれました。<br>まるで長いトンネルの先に希望の光が差し込んだような状況でした。
```

Comment:
Putting a br here does not work, because of the way the system gets these strings. The strings have to be "simple" with no HTML formatting, inside these translation files. @eishikawa